fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -72,6 +72,8 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
 
     private var readerModeAvailable = false
     private var reviewQualityCheckAvailable = false
+    private var translationsAvailable = false
```

Comment:
One question, do you think we should also lock this additionally behind the secret menu pref for now? 

I had a thought that it would be easy to set this to `true` while working and it slip into a PR. If it was missed in review, then it could start riding the trains as `true` accidentally.

Reply:
I don't think this should be necessary because we will want to eventually observe and determine whether or not translations are available. We would set that up via https://github.com/gabrielluong/firefox-android/commit/6685766c0c8d329388700e623f9ad0668a445aab#diff-d6281fdb81a5995139750c3bb8316ab4494ebfac844287ff0c691bf805661e81R231, which is already gated by the settings in `initTranslationsAction`

Reply:
Thought about this some more it might be a good idea to show this by default while we are working on the boilerplate UI.

Reply:
I like the idea of it being connected to secret settings. I can just see myself turning this on manually initially and forgetting about it.